IdentityServer/v7/docs/content/reference/models/license_usage_summary.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "License Usage Summary"
+description: "Get detailed information about your IdentityServer usage"
+date: 2025-01-07T12:00:00+02:00
+weight: 30
```

Comment:
@josephdecock @RolandGuijt could you verify what `weight` value this should be?

Reply:
Hmm, with weight 30, we end up right in between API Scopes and Resources, which probably should stay together? Maybe chuck it at the end, with 90?

Reply:
Updated to 90